### ISSUE TITLE
sigh. broken cli on npm...

### ISSUE THREAD

**Issue by moimikey** [moimikey/is-satire#9]
```
mshertzberg@forge:/Users/mshertzberg/Git/is-satire/src  git:(master) $ npm -g install is-satire
/Users/mshertzberg/.nvm/versions/node/v5.2.0/bin/is-satire -> /Users/mshertzberg/.nvm/versions/node/v5.2.0/lib/node_modules/is-satire/lib/cli.js
/Users/mshertzberg/.nvm/versions/node/v5.2.0/lib
└─┬ is-satire@1.0.8
  └─┬ babel-cli@6.3.17
    ├─┬ babel-core@6.4.0
    │ ├── babel-generator@6.4.2
    │ ├── babel-helpers@6.4.0
    │ ├─┬ babel-traverse@6.3.26
    │ │ └── globals@8.18.0
    │ ├── babel-types@6.4.1
    │ └── babylon@6.4.2
    └─┬ request@2.67.0
      ├─┬ http-signature@1.1.0
      │ └─┬ sshpk@1.7.2
      │   └── tweetnacl@0.13.3
      └─┬ mime-types@2.1.9
        └── mime-db@1.21.0

mshertzberg@forge:/Users/mshertzberg/Git/is-satire/src  git:(master) $ is-satire
mshertzberg@forge:/Users/mshertzberg/Git/is-satire/src  git:(master) $
```

**Comment by moimikey** [comment]
AHA. this is babel's fault.

**Comment by moimikey** [comment]
```
mshertzberg@forge:/Users/mshertzberg/Git/is-satire/lib  git:(master) $ ./cli.js
mshertzberg@forge:/Users/mshertzberg/Git/is-satire/lib  git:(master) $ ..
mshertzberg@forge:/Users/mshertzberg/Git/is-satire  git:(master) $ lib/cli.js
usage: is-satire http(s)://...
```

because babel is fugging doing this shiz in the piled lib:

```
var _construct = require('babel-runtime/core-js/reflect/construct');
var _construct2 = _interopRequireDefault(_construct);
[...]
```

**Comment by moimikey** [comment]
this was my stupid fault. closing. and fixed in 1.0.10

**Comment by moimikey** [comment]
turns out it was the json paths that were failing and i never `catch`'ed my promise :(... like an ametuer.
